src/dom-to-image-more.js

```diff
@@ -238,9 +291,22 @@
             });
 
         function newCanvas(node, scale) {
+            let width = options.width || util.width(node);
```

Comment:
So here's the big fix for various unsized elements... we get the sizes from `options` like before, or calculated from the node (like before)... but NOW we also handle `NaN` or `0` values as missing settings and apply the official specification for inline elements (e.g. 300x150 default). This makes `svg`'s of a `span` (or a text node wrapping in one) have some width and height so they will convert to an image correctly instead of throwing.